magpie/setup/magpie-setup-project-zeppelin

```diff
@@ -153,9 +153,18 @@ then
     phoenix_jars="${PHOENIX_HOME}/phoenix-${PHOENIX_VERSION}-server.jar,$phoenix_jars"
     phoenix_jars="${PHOENIX_HOME}/phoenix-spark-${PHOENIX_VERSION}.jar,$phoenix_jars"
     phoenix_jars="${PHOENIX_HOME}/phoenix-core-${PHOENIX_VERSION}.jar,$phoenix_jars"
-    phoenix_jars="${PHOENIX_HOME}/phoenix-server-${PHOENIX_VERSION}.jar,$phoenix_jars"
-    phoenix_jars="${PHOENIX_HOME}/phoenix-server-client-${PHOENIX_VERSION}.jar,$phoenix_jars"
-    zeppelin_jars="${zeppelin_jars},${phoenix_jars}"
+    
+    Magpie_vercomp ${PHOENIX_VERSION} 4.8.0
```

Comment:
Could you add ```# Returns 0 for ==, 1 for $1 > $2, 2 for $1 < $2``` at the top of the vercomp.  B/c Magpie_vercomp is not intuitive, I try to add it everywhere I use it.